src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends FigureForOther {
```

Comment:
rewrite code accordingly
```suggestion
public class IsoscelesTrapezoid extends Figure {
```